.gitignore

```diff
@@ -8,4 +8,4 @@
 /Infra.toml
 /*.pem
 /keys
-/roles
+/roles
```

Comment:
There's still a change in this file, removing the EOL.  Can we remove this file from the commits entirely?

Reply:
Oops sorry. Will fix.